vuepress/docs/next/tutorials/solution/pda-tutorial.md

```diff
@@ -56,6 +56,42 @@ ent ecr deploy --repo="https://github.com/entando-samples/entando-process-driven
 
 ## Configuration
 
+::: warning
+---
+
+<details><summary>Workaround for PDA connections on OpenShift 4.8</summary>
+
+The PDA Plugin connections cannot be configured for RedHat OpenShift 4.8 implementations of Entando 7 due to missing rules on the entando-plugin Role. To circumvent a connections error, add the following definition to `namespace-resources.yaml`:
```

Comment:
Minor change, my notes were not clear. "To circumvent a connections error, update the `entando-plugin` Role by adding the missing rules below."

I'm leaving it a little vague since you could do this with kubectl, k9s, or in the OpenShift UI like I did. This requires a bit of k8s knowledge of course.

I know for sure we need this on the 7.0 codebase but I'm unsure about 7.1 until we test. Maybe tweak this, add it to 7.0, and then add a note to our 7.1 TODOs to review later?